### ISSUE TITLE
jsx-indent rule on short-circuit evaluation renders ?

### ISSUE THREAD

**Issue by gomesalexandre** [yannickcr/eslint-plugin-react#1765]
Is there a rule for enforcing indentation in the case of short-circuit evaluated components ?

A common pattern in React is to use short-circuit evaluation to conditionnally render components. 

Take this bit of code, for example : 
```javascript          
            {this.props.visible.requestModal &&
            <RequestModal
              newRequest={newRequest}
            />
            }
```

which does not produce errors or warnings.

I would like for it to be indented this way i.e 2 spaces chars on the newline after the left element

 ```javascript
          {this.props.visible.requestModal &&
            <RequestModal
              newRequest={newRequest}
            />
          }
```

Is there a rule to enforce such indent either with jsx-indent or with another rule ? 

Thanks

**Comment by ljharb** [comment]
If you enforce that multiline jsx has wrapping parens, then the indent rules can ensure what you want.
